Autocoders/Python/src/fprime_ac/utils/Logger.py

```diff
@@ -46,7 +46,7 @@ def connectOutputLogger(file=None, logger_name="output"):
     Function that connects the output logger.
     This is always enabled and hardwired to
     generate logging.INFO level messages only.
-    @params  file : Output file to store these messages into.
+    @param   file : Output file to store these messages into.
```

Comment:
this appeared to be errant/inconsistent with prevailing use in this file/repository (and was caught by a new forbidden pattern).